align/compiler/compiler.py

```diff
@@ -185,7 +185,7 @@ def compiler_output(input_ckt, lib_names , hier_graph_dict, design_name:str, res
                 logger.debug(f"Created new lef for: {block_name} {lef_name}")
                 #Multiple instances of same module
                 if 'inst_copy' in attr:
-                    for nm in hier_graph_dict.keys():
+                    for nm in list(hier_graph_dict.keys()):
                         if nm == lef_name + attr['inst_copy']:
                             hier_graph_dict[block_name] = hier_graph_dict.pop(nm)
                     graph.nodes[node]["inst_type"]=block_name
```

Comment:
@kkunal1408 can I get you to verify this change is okay? You were modifying a dictionary while iterating over it. This is not supported by Python though a runtime check was only added in Python 3.8.

Reply:
Yes, the change is fine. I already have this change in another pr.

Reply:
Great!